modules/ROOT/pages/endpoint-as2-send.adoc

```diff
@@ -51,17 +51,21 @@ Create AS2 endpoints by configuring these settings on the partner or third-party
 |Your partner’s or third-party connection's AS2 URL to which outbound EDI transmissions are sent.
 |x
 
-|Authentication type
-a|Authentication type used to access the endpoint. Allowed values:
+|Certificate (TLS context)
+a| (Applies when the *Partner AS2 URL* field starts with `https`) SSL certificate. Values are:
 
-* None
+* `Use CA-signed certificates`
++
+Uses a certificate that was issued and signed by a certificate authority (CA). With this option, you don't need to upload an additional certificate.
+* `Use a self-signed certificate`
++
+Uses a certificate that was not issued or signed by a CA. With this option, you must either select an existing certificate or upload a new certificate in `.pem` format.
 +
-Does not require backend applications to authenticate when connecting to this endpoint.
+* `Allow insecure connection`
 +
-* Basic
+Allows an insecure connection that doesn’t perform certificate validation. 
 +
-Requires backend applications to use the username and password provided by your partner to connect to this endpoint.
-Enter this information in the *Username* and *Password* fields.
+Using this option is not recommended in production environments. Use it only during testing.
```

Comment:
**Don't use this option in** production environments. Use it only during testing.